### ISSUE TITLE
layoutXLM experiment on XFUN zh, I can't reproduce the result of re task

### ISSUE THREAD

**Issue by feather820** [microsoft/unilm#359]
**Describe**
Model I am using (UniLM, MiniLM, LayoutLM ...):
I am using layoutXLM to reproduce the result of XFUN dataset of zh, I got the mismatch result of RE task, I got 0.66, but the paper indicate it should be 0.70, I am using the same code,did't change anything.

**Comment by Ariel19977** [comment]
> **Describe**
> Model I am using (UniLM, MiniLM, LayoutLM ...):
> I am using layoutXLM to reproduce the result of XFUN dataset of zh, I got the mismatch result of RE task, I got 0.66, but the paper indicate it should be 0.70, I am using the same code,did't change anything.

Hello , I am using layoutXLM ro reproduce the result of XFun dataset of zh , but I have a problem with data download and its slowness. Could you share your command for reference？Very Thankful!

**Comment by wolfshow** [comment]
> **Describe**
> Model I am using (UniLM, MiniLM, LayoutLM ...):
> I am using layoutXLM to reproduce the result of XFUN dataset of zh, I got the mismatch result of RE task, I got 0.66, but the paper indicate it should be 0.70, I am using the same code,did't change anything.

You may need to fine-tune the model with more epochs.

**Comment by zhangxgu** [comment]
I also meet this problem. The  Relation Exaction performance on zh data is about 0.64 with the hyper-paramters in paper. However, the training loss is 0.03 while the val loss is 0.16 after 1000 iterations. @wolfshow  If more iterations are performed, the model may be overfitted. Can the author give us the hyperparamters to get the performances on XFUN zh relation task?

**Comment by ranpox** [comment]
Hi all,
I just updated the README and the RE module in this commit https://github.com/microsoft/unilm/commit/11b67b198a599e861f5185f63c159ebc5ab1ecaa. Please pull the latest `layoutlmft` code and check if it works.
Thanks.
